### PR TITLE
Consume build tools 00106, which adds support for auto test packages

### PR DESCRIPTION
**Pull request by FiveTimesTheFun**
Consume build tools 00106, which adds support for automatically
generating test nuget packages.

### CODE REVIEW

**Comment by FiveTimesTheFun** [comment]
Goes with https://github.com/dotnet/buildtools/pull/307

File: dir.props [hunk 1]
```diff
@@ -41,6 +42,7 @@
     <!-- Input Directories -->
     <PackagesDir Condition="'$(PackagesDir)'==''">$(ProjectDir)packages/</PackagesDir>
     <ToolsDir Condition="'$(ToolsDir)'==''">$(PackagesDir)Microsoft.DotNet.BuildTools.$(BuildToolsVersion)/lib/</ToolsDir>
+    <NuProjDir Condition="'$(NuProjDir)'==''">$(PackagesDir)NuProj.$(NuProjVersion)/tools/</NuProjDir>
```
Comment (weshaggard) [inline dir.props]: I see this is where NuProjDir is getting set. I'm not sure if I like the fact this this is required to be setup and restored by all the consuming repo's of BuildTools. We should probably include the library directly in the BuildTools package if we want to depend on it.

Comment (FiveTimesTheFun) [inline dir.props]: Thanks Wes, will reply over at https://github.com/dotnet/buildtools/pull/307.

**Comment by stephentoub** [comment]
Closing this out, as buildtools has already progressed past 106.

**Pull request closed without merging**